.rubocop.yml

```diff
@@ -108,6 +108,14 @@ Style/TrailingWhitespace:
 Style/ZeroLengthPredicate:
   Enabled: true
 
+
+# Check for array literals made up of word-like strings, that are not using the %w() syntax
+Style/WordArray:
+  EnforcedStyle: percent
+  MinSize: 3
```

Comment:
Why setting `MinSize` to 3? Was before like that, but we can change it now to 0, or remove it, as 0 is the default. :wink: